spock-specs/src/test/groovy/org/spockframework/runtime/ErrorSpecNodeSpec.groovy

```diff
@@ -0,0 +1,23 @@
+package org.spockframework.runtime
+
+import org.junit.platform.engine.TestDescriptor
+import org.junit.platform.engine.UniqueId
+import spock.lang.Specification
+
+class ErrorSpecNodeSpec extends Specification {
```

Comment:
I would add an integration test in `spock-testkit`

Reply:
I'd be happy to if you tell me how.
I tried to write a failing test, but was not able to.
The `EmbeddedSpecRunner` uses `EngineTestKit.engine("spock").selectors(selector).execute()` which does not come through the problematic part which is `org.junit.platform.launcher.core.DefaultLauncher#discoverRoot` that calls the `prune` method on the `Root` which then starts the recursive `prune` call on the test descriptors.

Reply:
Maybe @marcphilipp can help here, but I did not mean `EmbeddedSpecRunner` I meant a test in the `spock-testkit` module.

Reply:
Maybe I misunderstood then?
I just looked at `SpockHelloWorldTest` there and it uses the same call-chain to trigger a test.
What is the sense of the `spock-testkit` module?
What is the special thing about it?

Reply:
Maybe I misunderstood then?
I just looked at `SpockHelloWorldTest` there and it uses the same call-chain to trigger a test.
What is the sense of the `spock-testkit` module?
What is the special thing about it?